### DISCUSSION TITLE
AddHostOjectToScript caching

### DISCUSSION THREAD

**Discussion by Antix-Development** [MicrosoftEdge/WebView2Feedback#4123] [Q&A] [answered]
I read somewhere that caching host objects like `window.chrome.webview.hostedObjects` can cause issues. Is this actually true? I'd really prefer to cache them if it's safe.

Thanks, and really enjoying WebView2 so far :)

**Comment by victorhuangwq** [comment]
@LiangTheDev could you help @Antix-Development out?

**Comment by LiangTheDev** [comment]
You should be able to cache webview.hostObjects. Where did you read that and what are the issues mentioned?

**Comment by Antix-Development** [comment]
Thanks, I've been using it for a couple of days without issues.

I read about it on https://weblog.west-wind.com/posts/2021/Jan/26/Chromium-WebView2-Control-and-NET-to-JavaScript-Interop-Part-2

**Comment by LiangTheDev** [comment, marked as answer]
It seems that the blog doesn't have details of the issues. Anyway, we are not aware or expect any issues.

**Comment by Antix-Development** [comment]
Yes, the blog only mentions the issue but doesn't have any more detail than that.
